borrowings/models.py

```diff
@@ -0,0 +1,35 @@
+import datetime
+
+from django.conf import settings
+from django.db import models
+from books.models import Book
+
+
+def calculate_expected_return_date(borrow_days: int) -> datetime:
+
+    current_date = datetime.date.today()
+
+    return current_date + datetime.timedelta(days=borrow_days)
```

Comment:
it's necessary specify a variable `borrow_days ` if it's always the same number? you can solve it in one line

Reply:
Yes, best practice :)